modules/administration-guide/pages/installing-che-on-openshift-using-the-web-console.adoc

```diff
@@ -13,6 +13,16 @@ This section describes how to install {prod-short} using the OpenShift web conso
 
 * An OpenShift web console session by a cluster administrator. See link:https://docs.openshift.com/container-platform/{ocp4-ver}/web_console/web-console.html[Accessing the web console].
 
+[IMPORTANT]
+====
+Ensure that there are no `CheCluster` API by running the following command:
+[source,subs="+quotes,attributes"]
+----
+$ {orch-cli} get crd checlusters.org.eclipse.che
```

Comment:
This procedure uses OpenShift web console. Is there a GUI alternative to running this CLI command? If so, we can incorporate it into the procedure as a step in web console.